fern/pages/docs/components/code-blocks.mdx

```diff
@@ -1,10 +1,258 @@
+a `maxLines` attribute after the language identifier. The 
+`maxLines` attribute should be a number representing the maximum 
+number of lines to display. By default, the code block will display up to 20 lines.
+
+<Tabs>
+    <Tab title="Example">
+        ```python maxLines=10
+        def is_prime(num):
+            """Check if a number is prime."""
+            if num <= 1:
+                return False
+            for i in range(2, num):
+                if num % i == 0:
+                    return False
+            return True
+
+        start = 10
+        end = 50
+
+        print(f"Prime numbers between {start} and {end} are:")
+
+        prime_numbers = []
+
+        for num in range(start, end+1):
+            if is_prime(num):
+                prime_numbers.append(num)
+
+        for prime in prime_numbers:
+            print(prime)
+        ```
+    </Tab>
+    <Tab title = "Markdown">
+        ````markdown maxLines=10
+        ```python maxLines=10
+        def is_prime(num):
+            """Check if a number is prime."""
+            if num <= 1:
+                return False
+            for i in range(2, num):
+                if num % i == 0:
+                    return False
+            return True
+
+        start = 10
+        end = 50
+
+        print(f"Prime numbers between {start} and {end} are:")
+
+        prime_numbers = []
+
+        for num in range(start, end+1):
+            if is_prime(num):
+                prime_numbers.append(num)
+
+        for prime in prime_numbers:
+            print(prime)
+        ```
+        ````
+
+        <Note>
+        To disable the default 20 lines limit, you can set `maxLines` to `0`, or add the `fullHeight` attribute after the
+        language identifier.
+        </Note>
+    </Tab>
+</Tabs>
+
+
+## Combining Props
+
+You can combine the `title`, `highlight`, `focus`, and `maxLines` 
+props to create a code block with a title, highlighted lines, 
+and a maximum height.
+
+<Tabs>
+    <Tab title="Example">
+        ```javascript title="Hello, World!" {2-4} maxLines=5
+        console.log("Line 1");
+        console.log("Line 2");
+        console.log("Line 3");
+        console.log("Line 4");
+        console.log("Line 5");
+        console.log("Line 6");
+        console.log("Line 7");
+        console.log("Line 8");
+        console.log("Line 9");
+        console.log("Line 10");
+        ```
+    </Tab>
+    <Tab title = "Markdown">
+        ````markdown maxLines=5
+        ```javascript title="Hello, World!" {2-4} maxLines=5
+        console.log("Line 1");
+        console.log("Line 2");
+        console.log("Line 3");
+        console.log("Line 4");
+        console.log("Line 5");
+        console.log("Line 6");
+        console.log("Line 7");
+        console.log("Line 8");
+        console.log("Line 9");
+        console.log("Line 10");
+        ```
+        ````
+    </Tab>
+</Tabs>
+
+
+## Tabbed Code Blocks
```

Comment:
Can we remove the `Aside` and add it to this section like the rest of the snippets

<img width="726" alt="Screenshot 2024-05-07 at 5 33 41 PM" src="https://github.com/fern-api/fern/assets/10870189/05297e82-8763-4f50-8a7a-1e35a27bd20b">